examples/example-tls/README.md

```diff
@@ -19,6 +19,9 @@ in the
 `build/install/example-tls/bin/` directory that run the example. The
 example requires the server to be running before starting the client.
 
+
+## How to run
```

Comment:
Level 2 heading not needed. If added, "To build the example" above should also be a heading of the same level.

Reply:
Done.